src/xdg-autostart-generator/xdg-autostart-generator.c

```diff
@@ -44,38 +44,42 @@ static int enumerate_xdg_autostart(Hashmap *all_services) {
         STRV_FOREACH(path, autostart_dirs) {
                 _cleanup_closedir_ DIR *d = NULL;
 
+                log_debug("Scanning autostart directory \"%s\"…", *path);
                 d = opendir(*path);
                 if (!d) {
-                        if (errno != ENOENT)
-                                log_warning_errno(errno, "Opening %s failed, ignoring: %m", *path);
+                        log_full_errno(errno != ENOENT ? LOG_DEBUG : LOG_WARNING, errno,
```

Comment:
Previously this was logged at warning level for !ENOENT, but now it's all logged at debug level apart from ENOENT which is warning, is that intended?

Reply:
Oops.